### PR TITLE
SonarCloud

### LINKED ISSUE #19: [SonarCloud] Connect SonarCloud to the project
## Description
- register SonarCloud account
- connect SonarCloud to the project
- set settings to run SonarCloud check when you push or create pull request to main and develop branches
- set custom quality gate with test coverage equals to 70%
## Acceptance Criteria
- [ ] SonarCloud should be connected and run every time there is a push to main/develop branches or pull request is made to main/develop 
- [ ] test coverage should be 70%

### PR DESCRIPTION
**Pull request by HalynaProlubshchykova**
Connection to SonarCloud

### CODE REVIEW

**Review by olehpona — COMMENTED** [review]
looks good!

**Review by tufned — APPROVED** [review]

**Review by MrAK-dev — APPROVED** [review]
Code looks good!

**Review by SachkovaYuliia — APPROVED** [review]

**Review by olehpona — APPROVED** [review]
looks good!

File: sonar-project.properties [hunk 1]
```diff
@@ -0,0 +1,18 @@
+sonar.projectKey=WebUI-Project-Based-07-vk_BackEnd
+sonar.organization=webui-project-based-07-vk
+
+sonar.projectName=BackEnd
+
+# Path is relative to the sonar-project.properties file. Replace "\" by "/" on Windows.
+sonar.sources=.
+sonar.exclusions=**/src/tests/**/*, **/build-story/*
+sonar.coverage.exclusions=**/src/constants/*, **/src/tests/**/*, **/*.styles.ts
```
Comment (vlad-khrychov) [inline sonar-project.properties]: Do we have styles on back end part?
```suggestion
sonar.coverage.exclusions=**/src/constants/*, **/src/tests/**/*
```

File: sonar-project.properties [hunk 2]
```diff
@@ -0,0 +1,18 @@
...
+
+sonar.tests=./src/tests
```
Comment (vlad-khrychov) [inline sonar-project.properties]: In this repo tests are placed in folder `test`.
```suggestion
sonar.tests=./src/test
```

**Review by vlad-khrychov — CHANGES_REQUESTED** [review]

File: sonar-project.properties [hunk 3]
```diff
@@ -0,0 +1,18 @@
+sonar.projectKey=WebUI-Project-Based-07-vk_BackEnd
+sonar.organization=webui-project-based-07-vk
+
+sonar.projectName=BackEnd
+
+# Path is relative to the sonar-project.properties file. Replace "\" by "/" on Windows.
+sonar.sources=.
+sonar.exclusions=**/src/test/**/*
```
Comment (vlad-khrychov) [inline sonar-project.properties]:
```suggestion
sonar.exclusions=**/src/test/**/*, **/src/consts/*, docs/**
```

**Review by vlad-khrychov — CHANGES_REQUESTED** [review]

**Review by vlad-khrychov — APPROVED** [review]

**Pull request merged**